packages/ember-glimmer/lib/environment.js

```diff
@@ -252,7 +254,7 @@ export default class Environment extends GlimmerEnvironment {
   // a Compiler can wrap the template so it needs its own cache
   getCompiledBlock(Compiler, template, owner) {
     let compilerCache = this._compilerCache.get(Compiler);
-    return compilerCache.get(template, owner);
+    return compilerCache.get({ template, owner });
```

Comment:
kk, I also thought about making the cache pass through arbitrary numbers of positional arguments